DTS-STN/dts-rmp
# 278 comment date

# Description

- update date format to display time
- fix hard-coded user and date 

## Test Instructions

1. go to search engagement page
2. select any engagement to view detail
3. add new comments, date and time should be displayed in the format of 'YYYY-MM-DD 10:00am'.
4. user and date are no longer hard-coded. 

## Helps

Got help from Beniot and Jordan 

## Reviewers

@will0684 @MarcoGoC @shewood @nghe0001 @bcloutier7 @andr0272 

## Checklist
